TESTING.md

```diff
@@ -0,0 +1,76 @@
+- [ ] Test CLI update on Linux via YUM
+- [ ] Test CLI update on Linux via curl script
+- [ ] Test CLI update on Windows via chocolatey
+- [ ] Test CLI update on Windows via manual download
+- [ ] Test server installation via CLI with Docker Compose and no demo API
+- [ ] Test server installation via CLI with Docker Compose and demo API
+- [ ] Test server installation via CLI with Kubernetes and no demo API
+- [ ] Test server installation via CLI with Kubernetes and demo API
+- [ ] Double check [Detailed installation](https://docs.tracetest.io/getting-started/detailed-installation) doc and see if everything is documented correctly
+
+- [ ] Test Tracetest setup with [No tracing example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-no-tracing) by checking the last run for our CI on **Test examples** step
+- [ ] Test Tracetest setup with [Jaeger example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-jaeger) by checking the last run for our CI on **Test examples** step
+- [ ] Test Tracetest setup with [OpenSearch example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-opensearch) by checking the last run for our CI on **Test examples** step
+- [ ] Test Tracetest setup with [Tempo example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-tempo) by checking the last run for our CI on **Test examples** step
+
+- [ ] Test Tracetest setup with [Amazon X-Ray example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-amazon-x-ray)
+- [ ] Test Tracetest setup with [Datadog example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-datadog)
+- [ ] Test Tracetest setup with [Elastic APM example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-elasticapm)
+- [ ] Test Tracetest setup with [Lightstep example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-lightstep)
+- [ ] Test Tracetest setup with [New Relic example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-new-relic)
+- [ ] Test Tracetest setup with [SignalFX example](https://github.com/kubeshop/tracetest/tree/main/examples/tracetest-signalfx)
+
+- [ ] Check if our End-to-end pipelines tested correctly our main features:
+  - [ ] The `WebUI End-to-end tests` Github action should run without errors
+  - [ ] The `Tracetesting API Server` Github action should run without errors
+
+- [ ] Test specific features added/changed on this release on a local installation or in a Kubernetes installation:
+  - [ ] Feature 1
+  - [ ] Feature 2
+
+## Automatic Tests
+
+Today Tracetest has 3 main components: a WebUI, a CLI and an API Server.
+
+### Web UI
+
+- **Unit tests**: Run by executing `npm test` on `./web` folder
+- **End-to-end tests**: Run using [cypress](https://www.cypress.io/) against a temporary Tracetest created on Kubernetes. 
+
+### CLI
+
+- **Unit tests**: Run by executing `make test` on `./cli` folder
+
+### API Server
```

Comment:
```suggestion
### Server
```

Reply:
It makes sense because today, our Server hosts the API, the OTLP internal server, and the workers, right? I'm changing the name here and in other parts of the document.